src/openssl.cnf.sample

```diff
@@ -63,5 +73,5 @@ init = 1
 #
 # PKEY_CRYPTO
 # - X25519, X448, ED25519, ED448
-default_algorithms = ALL
-#default_algorithms = PKEY_CRYPTO,RAND,RSA,DH,DSA,CIPHERS,DIGESTS
+
+default_algorithms = RSA,DH,DSA
```

Comment:
Patrick, don't you want RAND also enabled by default ?

Reply:
Good point, i ll change that.

Reply:
done.